code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -630,8 +630,8 @@ var/global/list/activated_medevac_stretchers = list()
 		user.visible_message("<span class='notice'>[user] [locked ? "locks" : "unlocks"] [src]'s interface.</span>",
 		"<span class='notice'>You [locked ? "lock" : "unlock"] [src]'s interface.</span>")
 	else if(istype(O,/obj/item/roller/medevac))
-		if(locked && !allowed(user))
-			to_chat(user, "<span class='warning'>Access denied.</span>")
+		if(locked)
+			to_chat(user, "<span class='warning'>[src]'s interface is locked! Only a Squad Leader, Corpsman, or Medical Officer can unlock it now.</span>")
```

Comment:
Keep the `Access denied.` feedback, as it's more generic and maintainable.
There's more people who can unlock it, like synths, captains, ERTs and whatnot, and it's not really possible to list them all or keep up with access changes elsewhere.